scenarios/throughput_stress_test.go

```diff
@@ -18,9 +18,10 @@ func TestThroughputStress(t *testing.T) {
 	runID := fmt.Sprintf("tps-%d", time.Now().Unix())
 	taskQueueName := loadgen.TaskQueueForRun(scenarioName, runID)
 
-	env := workers.SetupTestEnvironment(t)
-	nexusEndpointName, err := env.CreateNexusEndpoint(t.Context(), taskQueueName)
-	require.NoError(t, err, "Failed to create Nexus endpoint")
+	env := workers.SetupTestEnvironment(t,
+		workers.WithExecutorTimeout(2*time.Minute),
```

Comment:
Admittedly, I'm not quite sure why it takes so long now.

There seem to be larger 8s jumps.

<img width="1270" height="1599" alt="Image" src="https://github.com/user-attachments/assets/92f7e075-2c72-4104-9d82-fda85fb68b21" />

Reply:
Okay, I narrowed it down to the `createSelfUpdateWithPayloadAsLocal` local activity. But I'm not quite sure why. I left a TODO there now.